src/data.js

```diff
@@ -2064,6 +2064,26 @@ const pages = [
       'YouTuber',
     ],
   },
+  {
+    name: 'Seyhun Akyürek',
+    description:
+      'Senior Software Specialist. Swift, Objective-C, former Rubyist. Author of twitter-bootstrap-rails',
+    url: 'https://www.seyhunakyurek.com/uses/',
```

Comment:
This URL 404s, let me know when you've addressed this.

Reply:
Hello,  addressed url is fixed.